cms/server/admin/handlers/user.py

```
@@ -29,35 +29,187 @@
+            attrs["date_of_birth"] = None
+        else:
+            try:
+                parsed_date = date.fromisoformat(date_of_birth_str)
+                # Validate date is not in the future
+                if parsed_date > date.today():
+                    raise ValueError("Date of birth cannot be in the future")
+                # Add 120-year lower bound check
+                today = date.today()
+                min_date = date(today.year - 120, today.month, today.day)
+                if parsed_date < min_date:
+                    raise ValueError("Date of birth cannot be more than 120 years ago")
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)